src/frontend/src/flows/addDevice/manage/addDeviceSuccess.ts

```diff
@@ -0,0 +1,68 @@
+import { html, render } from "lit-html";
+import { mainWindow } from "../../../components/mainWindow";
+import { I18n } from "../../../i18n";
+import { renderPage } from "../../../utils/lit-html";
+import copyJson from "./addDeviceSuccess.json";
+
+export type DeviceAlias = string;
+
+export interface AddDeviceSuccessTemplateProps {
+  deviceAlias: DeviceAlias;
+  onContinue: () => void;
+  i18n: I18n;
+}
+
+const addDeviceSuccessTemplate = ({
+  deviceAlias,
+  onContinue,
+  i18n,
+}: AddDeviceSuccessTemplateProps) => {
+  const { title, continue_to_home, explore } = i18n.i18n(copyJson);
+
+  const slot = html`<article>
+    <hgroup>
+      <h1 class="t-title t-title--main">${title}</h1>
+      <p class="t-lead">${explore}</p>
+    </hgroup>
+    <output class="c-input c-input--readonly t-vip t-vip--small"
+      >${deviceAlias}</output
+    >
+    <div class="l-stack">
+      <button
+        @click=${onContinue}
+        class="c-button c-button--primary"
+        id="continueToHome"
```

Comment:
```suggestion
        data-action="next"
```

or similar to avoid ids

Reply:
Yes, all new screens use `data-foo`. I just haven't had time to change old screens, my bad.